code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -13,6 +13,8 @@
 	var/active_offset = 0
 	/// What, if any, submap we render onto
 	var/map = ""
+	/// Due to a Byond bug relating to secondary maps and positional screen_loc arguments, we need to statically set to the loc
+	var/relay_loc_override = null
```

Comment:
could just call this relay_loc

Reply:
gotcha, i wanted to call it an override because if all the byond code worked properly then CENTER would work and this wouldn't be necessary. i'll change it though